exampleSite/content/blog/v0.10.zh-cn.md

```
@@ -0,0 +1,247 @@
+  - name: imfing
+    link: https://github.com/imfing
+    image: https://github.com/imfing.png
+tags:
+  - Release
+---
+
+Hextra v0.10.0 是一个重大版本更新，包含多项新功能、架构升级和使用体验优化。
+
+<!--more-->
+
+本次更新还包含了来自 10 位 [新贡献者](#contributors) 的代码提交，并解决了社区长期期待的功能需求。
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)